ts/defs.bzl

```diff
@@ -222,19 +222,18 @@ def ts_project(
 
         supports_workers: Whether the worker protocol is enabled.
             To disable worker mode for a particular target set `supports_workers` to `False`.
-            Worker mode can be controlled as well via `--strategy` and `mnemonic` and  using .bazelrc.
 
-            Put this in your .bazelrc to disable it globally: `build --strategy=TsProject=sandboxed`
+            Worker mode can be disabled workspace wide by using the `--@aspect_rules_ts//ts:supports_workers` flag. 
+            To disable worker mode globally, insert `build --@aspect_rules_ts//ts:supports_workers=false` into the .bazelrc.
+
+            Alternatively, worker mode can be controlled via `--strategy`.
+            To disable worker mode globally via `--strategy` insert `build --strategy=TsProject=sandboxed` into the .bazelrc.
```

Comment:
isn't this the choice that doesn't work correctly though?

Reply:
when #264 lands, using `--strategy=TsProject=sandboxed` will be identical to having `supports_workers = False`

Reply:
even without #264, `--strategy=TsProject=sandboxed` works correctly with minor changes in log output.